CHANGELOG.md

```diff
@@ -3,7 +3,41 @@
+- rest: Allow users to configure the AWS STS domain when using Web Identity Credentials ([#5915](https://github.com/open-policy-agent/opa/issues/5915)) authored by @johanfylling
+- status: Add an OPA environment information Gauge to Prometheus metrics to capture information like OPA version ([#5852](https://github.com/open-policy-agent/opa/issues/5852)) authored by @jmoghisi
+- server: Add ability to configure Unix socket permissions if OPA is listening on a Unix socket ([#5888](https://github.com/open-policy-agent/opa/pull/5888)) authored by @ashutosh-narkar
+- loader: Allow extensions to the `loader` package that provide ability to register handlers for certain file extension. This feature is currently **EXPERIMENTAL**  ([#5940](https://github.com/open-policy-agent/opa/pull/5940)) authored by @srenatus
+
+### Topdown and Rego
+
+- new builtin `crypto.x509.parse_keypair`: Returns a key pair from a pair of PEM or base64 encoded data. See [the documentation on the new built-in](https://www.openpolicyagent.org/docs/v0.53.0/policy-reference/#builtin-crypto-cryptox509parse_keypair) for all the details. ([#5853](https://github.com/open-policy-agent/opa/issues/5853)) authored by @volck.
+- ast: Abort query evaluation if the compiler has errors. These errors will be exposed via the Status API if enabled ([#5947](https://github.com/open-policy-agent/opa/issues/5947)) authored by @johanfylling
+- `io.jwt.decode_verify`: Fix issue where token verification succeeded in case where `iss` constraint was required but token did not contain it ([#5850](https://github.com/open-policy-agent/opa/issues/5850)) authored by @AleksanderBrzozowski
+- wasm: Fix memory leaks in WASM when when incrementally adding or removing data ([#5785](https://github.com/open-policy-agent/opa/issues/5785)) and ([#5901](https://github.com/open-policy-agent/opa/issues/5901)) authored by @ctelfer-sophos
+- `http.send`: Add a new option to the `http.send` input object which allows policy authors to specify a retry count for executing a HTTP request. Retries are performed with an exponential backoff delay ([#5891](https://github.com/open-policy-agent/opa/pull/5891)) authored by @ashutosh-narkar
+- ast: Fix issue with `_` matching only scalars in rule indexing for arrays ([#5916](https://github.com/open-policy-agent/opa/pull/5916)) authored by @jaspervdj
+- rego: Allow for extending the Rego evaluation targets with plugins ([#5939](https://github.com/open-policy-agent/opa/pull/5939)) authored by @srenatus
+
+### Miscellaneous
+
+- Add PITS Global Data Recovery Services to ADOPTERS.md (authored by @pheianox)
+- Avoid unnecessary byte/string conversion by using alternative functions/methods ([#5944](https://github.com/open-policy-agent/opa/pull/5944)) authored by @Juneezee
+- Fix [CVE-2022-3517](https://github.com/advisories/GHSA-f8q6-p94x-37v3) by removing the affected code ([#5941](https://github.com/open-policy-agent/opa/pull/5941)) authored by @testwill
```

Comment:
That suggests that there had been a vulnerability. It has been dead code, so let's call it "false positive finding" to be clear, maybe?

Reply:
Updated text